runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1173,6 +1173,78 @@ NNFW_STATUS nnfw_session::set_backends_per_operation(const char *backend_setting
   return NNFW_STATUS_NO_ERROR;
 }
 
+NNFW_STATUS nnfw_session::train_set_traininfo(const nnfw_train_info *info)
+{
+  if (not isStateModelLoaded())
+  {
+    std::cerr << "Error during nnfw_session::train_set_traininfo : invalid state" << std::endl;
+    std::cerr << "nnfw_session::train_set_traininfo must be called in MODEL_LOADED state"
```

Comment:
```suggestion
```

Invalid state error is enough. The `MODEL_LOADED` state is internal status. So the user will not be able to understand what it means.

Reply:
Updated :)